hurumap_tz/templates/profile/profile_detail.html

```diff
@@ -358,8 +358,12 @@ <h1>Primary School Teachers</h1>
                  {% include 'profile/_blocks/_stat_list.html' with stat=primary_school_teachers.number_of_teachers stat_type='number' %}
              </div>
             <div class="column-half" id="chart-pie-primary_school_teachers-primary_school_teachers" data-stat-type="percentage" data-chart-title="Distribution of primary school teachers based on gender(2016)" data-initial-sort="-value"></div>
-
         </section>
+        <!--<section class="clearfix stat-row">
+          <div class="column-half">
+              {% include 'profile/_blocks/_stat_list.html' with stat=primary_school_desks.number_of_desks stat_type='number' %}
+          </div>
+      </section>-->
```

Comment:
Would this be needed if it's commented out?

Reply:
Yes, It's pending a data discrepancy fix and some new stats